core/modules/taxonomy/taxonomy.module

```diff
@@ -1852,6 +1853,49 @@ function taxonomy_field_widget_form(&$form, &$form_state, $field, $instance, $la
   return $element;
 }
 
+/**
+ * Implements hook_field_widget_options_buttons_form_alter().
+ */
+function taxonomy_field_widget_options_buttons_form_alter(&$element, &$form_state, $context) {
```

Comment:
We should be using a `hook_field_widget_form_alter()` instead . Please see https://github.com/bugfolder/backdrop/pull/7/files.